GravityParticle.h

```diff
@@ -311,6 +310,9 @@ class GravityParticle : public ExternalGravityParticle {
 public:
 	SFC::Key key;
 	Vector3D<double> velocity;
+    double dPy;
```

Comment:
Document this variable.  Look at dtGrav below for the doxygen friendly comment.
Also: change the indentation to 8 spaces to patch the tab.